corpus/templates/base.html

```diff
@@ -4,19 +4,109 @@
+    <meta name="twitter:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}">
+    <meta name="twitter:description"
+        content="{% if post.description %}{{ post.description|striptags|linebreaksbr|truncatechars:160 }}{% else %}{{ site_description }}{% endif %}">
+    <meta property="twitter:image:src"
+        content="{% static 'img/blog-image.png' %}">
+
+    <!-- Social: Facebook / Open Graph -->
+    <meta property="fb:app_id" content="{{ site.facebook_app_id }}" />
+    <meta property="og:url" content="{{ site.url }}{{ site.baseurl }}{{ page.url }}" />
+    <meta property="og:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}">
+    <meta property="og:image" content="{% static 'img/blog-image.png' %}">
+    <meta property="og:description"
+        content="{% if page.description %}{{ page.description|striptags|linebreaksbr|truncatechars:160 }}{% else %}{{ site.description }}{% endif %}">
+    <meta property="og:site_name" content="{{ site.title }}">
+
+    <!-- Social: Google+ / Schema.org  -->
+    <meta itemprop="name" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}" />
+    <meta itemprop="description"
+        content="{% if page.description %}{{ page.description|striptags|linebreaksbr|truncatechars:160 }}{% else %}{{ site.description }}{% endif %}">
+    <meta itemprop="image" content="{% static 'img/blog-image.png' %}" />
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
+    <meta name="theme-color" content="#ffffff">
+
+    <!-- Windows 8 Tile Icons -->
+    <meta name="application-name" content="{{site.username}}">
+    <meta name="msapplication-TileColor" content="#0562DC">
+    <meta name="msapplication-square70x70logo" content="smalltile.png" />
+    <meta name="msapplication-square150x150logo" content="mediumtile.png" />
+    <meta name="msapplication-wide310x150logo" content="widetile.png" />
+    <meta name="msapplication-square310x310logo" content="largetile.png" />
+
+    <!-- Android Lolipop Theme Color -->
+    <meta name="theme-color" content="#0562DC">
+
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Website Title -->
     <title>
         {% block title %} {% endblock %} | Corpus
     </title>
+
+    <!-- Linked Sheets needed -->
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
     <link rel="stylesheet"
           href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"/>
+    <link href="https://cdn.jsdelivr.net/npm/daisyui@4.4.6/dist/full.min.css" rel="stylesheet" type="text/css"/>
```

Comment:
Why are we fetching the full DaisyUI CSS? Isn't the JS Toolchain in place so that we don't need to import this?

Reply:
done